python/pyspark/ml/classification.py

```diff
@@ -454,7 +454,8 @@ class TreeClassifierParams(object):
     impurity = Param(Params._dummy(), "impurity",
                      "Criterion used for information gain calculation (case-insensitive). " +
                      "Supported options: " +
-                     ", ".join(supportedImpurities), typeConverter=TypeConverters.toString)
+                     ", ".join(supportedImpurities) + ". (Default: gini)",
```

Comment:
I think the rest of this PR can move ahead and get merged, so let's just revert this line and follow up later.